### ISSUE TITLE
Import external fonts

### ISSUE THREAD

**Issue by hesselbearda** [microsoft/vscode#81371]
**Feature Request**
I would like to be able to import external hosted fonts and use these in VS Code.

**Background Information**
I'm using a system with restricted permission rights so it doesn't let me install new fonts.

**Comment by Cyphexl** [comment]
It's hard to design an "import external fonts" option in the settings menu. Typically we'll need multiple style/weight variants for a single font family, but a setting option cannot hold a list.
The best option would probably be using CSS `@import` statements to import a stylesheet containing `@font-face` declarations.
I believe there are plugins like [vscode-custom-css](https://github.com/be5invis/vscode-custom-css) that already does that.

**Comment by hesselbearda** [comment]
Hi Cyphexl, thanks for your feedback. I do recognise that it's not easy but it's also not impossible :)

Before I wrote this feature request I tried out vscode-custom-css on a mac with full admin rights and on a windows machine with the restricted admin rights. The plugin didn't work as expected on both machines. Therefor I'm asking for native support.

Fun fact was that the creator of vscode-custom-css stated following: 

> Please, leave a review if you can so the VS Code Team can know that this is a very demanded feature and, maybe, they can then provide a proper way to extend the IDE regarding icons and customizations soon enough. ;D

Therefor I'm hoping there is enough demand to invest time to develop this awesome feature!
